la_palma_overview/__init__.py

```diff
@@ -75,17 +75,21 @@ def clock2img(rows, cols):
         fill='red'
     )
     try:
-        run_id = sfc.main_page().run_id
-        if run_id is not None:
-            run_str = 'Run {0: 3d}'.format(run_id)
-            w, h = d.textsize(run_str, font=font_run)
-            d.text(
-                ((cols - w) / 2, rows - h - 10),
-                run_str,
-                font=font_run,
-                anchor='top',
-                fill='red'
-            )
+        runs = sfc.observations(fallback=True).runs
+
+        if runs is not None:
+            last_run = runs[-1]
```

Comment:
could except if `runs = []`
not sure if this can even happen from the `sfc` side, but still ... if runs is list-like in can be empty.